src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -115,6 +115,7 @@ export type WalletConfig = {
          */
         unlocked_accounts: Array<string | number>;
       };
+      cliType: (string | number)[];
```

Comment:
TIL: a yargs option with `type: "array"` causes it to try to coerce args to a number, but if that fails, it will return a string. So `0x132456789...ABCDEF` results in a giant number! (i've fixed this in the latest code)